l10n_es_aeat_mod347/security/mod_347_security.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <data>
+        <record id="l10n_es_aeat_mod347_report_rule" model="ir.rule">
+            <field name="name">AEAT Mod 347 multi-company</field>
+            <field ref="model_l10n_es_aeat_mod347_report" name="model_id" />
+            <field eval="True" name="global" />
+            <field
+                name="domain_force"
+            >['|', ('company_id', '=', False), ('company_id', 'child_of', [user.company_id.id])]</field>
```

Comment:
Hay que adaptar esta regla al nuevo estilo de v13 y hacer script de migración. Mira la guía de migración:

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#tasks-to-do-in-the-migration